clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -570,10 +570,10 @@ getTripleBasedSYCLPostLinkOpts(const ArgList &Args,
   bool SplitEsimd =
       Args.hasFlag(OPT_sycl_device_code_split_esimd,
                    OPT_no_sycl_device_code_split_esimd, SplitEsimdByDefault);
-
-  // Symbol file and specialization constant info generation is mandatory -
+  if (!Args.hasArg(OPT_sycl_thin_lto))
+    PostLinkArgs.push_back("-symbols");
+  // Specialization constant info generation is mandatory -
```

Comment:
I do not know how and when this comment got here, I can handle this in a future commit.

Thanks

Reply:
sure, thx